source/project-stories/sh24.erb

```diff
@@ -1,37 +1,65 @@
 
 header:
   client: "SH:24"
-  introduction: "Unboxed transform NHS sexual health clinics to a digital service in 8 weeks"
+  introduction: "Transformation of NHS face-to-face sexual health clinic to digital service in 8 weeks"
   image: "/project-stories/sh24/sh24-post-it.jpg"
   image_description: "Sexual Health - how do we make it easier to access NHS services?"
 
 introduction:
-  title: "Digital healthcare innovation"
-  content: "SH:24 asked Unboxed to help them develop an online STI testing service. The vision was that a self service testing kit, ordered online could tackle long queues at inaccessible sexual health centres and dramatically increase public access to sexual health service."
+  title: "Online sexual health testing service to tackle high clinic demand"
+  content:
+    - "8 weeks from an idea to a working online service"
+    - "16 weeks from inception to NHS approved service"
+    - "95% of people who start the order process go through to completion"
+    - "Service to be rolled out to further areas in 2016"
   image: "/project-stories/sh24/kit-instructions.jpg"
   image_description: "Using a web prototype for instructions"
 
-statistics_title: "SH:24 in numbers"
-statistics:
-  - "8 weeks to successful prototype"
-  - "100+ users involved in design"
-  - "16 weeks from inception to NHS approved service"
-  - "1,300 test kits ordered in first 8 weeks"
-  - "95% starting an order online, complete it"
+statistics_1:
+  title: "SH:24 in numbers"
+  number: "100+"
+  text: "Users involved in the design"
 
-launch:
-  title: "Live service launched in 16 weeks"
-  content: "After only eight weeks, a user was able to order and complete an STI test at home without any clinical support. Another eight weeks later, a live service was launched to the public. This project is paving the way for transformation of other NHS services that are currently delivered face to face, in clinics."
-  image: "/project-stories/sh24/sti-test-kit.jpg"
-  image_description: "Opened SH:24 test kit"
+statistics_2:
+  number: "1,300"
+  text: "Test kits ordered in first 8 weeks"
+
+statistics_3:
+  title: "SH:24 in numbers since launch"
+  number: "5,200"
+  text: "Site visitors (70% new)"
+
+statistics_4:
+  number: "26,000"
+  text: "Page views"
+
+statistics_5:
+  number: "1,300"
+  text: "Tests orders"
+
+statistics_6:
+  number: "95%"
+  text: "People starting the order process through to completion"
+
+online_process:
+  title: "Taking the process from the doctor’s waiting room to online"
+  content: "Following a string of negative news headlines and a rise in sexual health STI rates in the Lambeth and Southwark areas, SH:24 came to us with an idea for a digital service: An online sexual health testing service for the residents of these areas. The vision was a self-service testing kit, ordered online, which could tackle long queues at inaccessible sexual health centers and dramatically increase public access to sexual health service."
```

Comment:
This content is too large for the box unfortunately: 
![Uploading Screen Shot 2016-01-04 at 11.20.18.png…]()